### PR TITLE
Report the failure rate of flush operations of the Elasticsearch exporter

### LINKED ISSUE #4830: ElasticExporter export metrics for bulk even when bulk is not flushed successfully
**Describe the bug**
During investigating an benchmark issue where elastic went full and was marked as read-only I was wondering why I can still see bulks in the metrics.

Looking at the code I found that the metrics are exported before the bulk is written. This means even if the bulk failed it is reported. https://github.com/zeebe-io/zeebe/blob/develop/exporters/elasticsearch-exporter/src/main/java/io/zeebe/exporter/ElasticsearchClient.java#L144

Looks like this:

elastic

**To Reproduce**
Just fill up elastic or disconnect elastic.

**Expected behavior**
Bulks are not reported in the metrics.

### PR DESCRIPTION
**Pull request by npepinpe**
## Description

This PR adds a new metric to visualize the failure rate of flush operations in the Elasticsearch exporter. Flush metrics are still reported on failure anyway as it can still be useful to have these metrics even on failure.

It looks like this in Grafana:

es-failed-flush

## Related issues

closes #4830 

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/zeebe-io/zeebe/compare/stable/0.24...develop?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/0.25`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the [release announcement](https://drive.google.com/drive/u/0/folders/1DTIeswnEEq-NggJ25rm2BsDjcCQpDape)

### CODE REVIEW

File: monitor/grafana/zeebe.json [hunk 1]
```diff
@@ -9517,6 +9517,101 @@
...
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "exemplar": true,
+              "expr": "rate(zeebe_elasticsearch_exporter_failed_flush_total{namespace=~\"$namespace\", pod=~\"$pod\"}[15s]) / rate(zeebe_elasticsearch_exporter_flush_duration_seconds_count{namespace=~\"$namespace\", pod=~\"$pod\"}[15s])",
+              "interval": "",
+              "legendFormat": "",
```
Comment (Zelldon) [inline monitor/grafana/zeebe.json:9570]: Could we add some better legendFormat? 
```suggestion
              "legendFormat": "{{pod}} Exporter {{partition}}",
```

**Review by ChrisKujawa — APPROVED** [review]
Thanks @npepinpe :+1: 

I was wondering whether we need such a large panel for this? Or do we want to have maybe next to it also a panel which shows the count of flushes as well?

**Comment by npepinpe** [comment]
So have them 2x2? As in:

----------------------
| Bulk | Memory |
|-------|-------------|
| Duration |  Failure Rate |

**Review by ChrisKujawa — APPROVED** [review]
Thanks :rocket:

File: monitor/docker-compose.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ services:
       - 9090:9090
 
   grafana:
-    image: grafana/grafana:6.7.5
+    image: grafana/grafana:7.5.2
```
Comment (Zelldon) [inline monitor/docker-compose.yml:22]: Can this break some of our panels ?

Comment (npepinpe) [reply monitor/docker-compose.yml:22]: I didn't notice anything, but possibly. Keep in mind though that we run 7.x in prod and in our zeebe-io cluster, so it should work fine.

**Comment by npepinpe** [comment]
bors merge

**Comment by npepinpe** [comment]
bors merge

**Pull request merged**